profile/README.md

```diff
@@ -0,0 +1,65 @@
+## Products:
+
+* [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
+![Hastlayer-logo](https://user-images.githubusercontent.com/68704777/171685373-7a64e9bf-b0fc-431d-a8d1-9027e0989b15.png)
+
+* [**DotNest** ](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
+![er6XI9B](https://user-images.githubusercontent.com/68704777/171690190-225d15a4-412d-40f4-b97f-63e192b4e596.png)
+
+## Our services:
+
+### Software development:
+
+Our team’s passion is developing software with
+care for clients who need help with their most
+extraordinary projects in the following frameworks:
+
+- Orchard Core.
+- Orchard CMS.
+- ASP.NET Core.
+- ASP.NET.
+
+### Consulting:
+
+We offer Orchard consulting in both Orchard editions,
+Orchard Core and Orchard CMS, including website
+and application development, and code review.
+
+### Training:
+
+Our team of dedicated experts is here to help groups
+and individuals who want to learn about Orchard
+or simply broaden their coding knowledge. Our
+[Orchard Dojo](https://orcharddojo.net/) website contains several training
+sessions to help beginners become experts.
+![Capturew](https://user-images.githubusercontent.com/68704777/172860375-ae1dc9a2-e46a-483a-ad5c-350a11ed732f.PNG)
+
+
+**We also contribute a vast amount of code, documentation, and learning resources to the open-source ecosystem. These materials are also available for our partners to use even after we have delivered a successful project.**
+
+## Our team
+Lombiq’s team consists of highly skilled experts from diverse backgrounds:
+
+![Capture](https://user-images.githubusercontent.com/68704777/172594376-9a506b57-4434-4536-95d7-2da33a5bd6fb.PNG)
+
+## Open-source and open hearts
+Apart from the software world we also try to help in other communities with our philanthropic activities. Most recently, we've delivered thousands of reusable masks to the Hungarian Charity Service of the Order of Malta during the coronavirus pandemic, as well as help for refugees of the war in Ukraine, and again.
+
+# Would you like to contribute to our projects?
+
+That would be really fantastic! If you've found one of our projects valuable and you'd like to contribute in any way please check out our [guidelines](https://lombiq.com/open-source-guidelines) and go ahead, we'd be very happy to welcome your efforts! Also, consider contributing, in-kind or otherwise, to the projects supported by us.
+
+You can also check our profile in [NuGet](https://www.nuget.org/profiles/Lombiq).
+
+Would you like to get in touch with us? Feel free to reach out to us through our Social media accounts:
+* [Twitter](https://twitter.com/Lombiq)
+* [YouTube ](https://www.youtube.com/c/LombiqTechnologies)
```

Comment:
```suggestion
* [YouTube](https://www.youtube.com/c/LombiqTechnologies)
```

Reply:
I updated the file according to the suggested changes. Please let me know if anything is missing. And Thank you for your patience